pkg/registry/apis/provisioning/export.go

```diff
@@ -68,7 +68,7 @@ func (c *exportConnector) Connect(ctx context.Context, name string, opts runtime
 				Push:       options,
 			},
 		})
-		if err != nil {
+		if err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
shouldn't this be a 409 Conflict? We didn't accept it per se. We could handle the 409 already exist in the frontend to give the users a nice message.